src/test/regress/input/multi_copy.source

```diff
@@ -577,15 +577,6 @@ ALTER USER test_user WITH nologin;
 
 -- reissue copy, and it should fail
 COPY numbers_hash FROM STDIN WITH (FORMAT 'csv');
```

Comment:
When I tried moving this to MX schedule, I also realized that we had to remove these lines. Though, I couldn't really understand why. Do you have an idea why we need to remove these input?

I think it is fine even if we don't understand, these `.source` tests are weird anyway.

Reply:
This was not just in `.source` tests. If I'm not mistaken, previously it was failing when we try to copy rows but now it fails before that. I think now the failure might be when we try to take a lock. So the actual rows are not even read.